### PR TITLE
launcher, dedicated: Rename wrong library name

### PR DESCRIPTION
**Pull request by yumirak**
Fix issue #5 and missing libvstdlib_client.so when launching dedicated server

### CODE REVIEW

File: dedicated_main/main.cpp
Comment (r-a-sattarov) [inline dedicated_main/main.cpp:1]: How about changing the name of this library also in these files?
- matchmaking/matchmaking_linux64_csgo.mak#L93
- matchmaking/matchmaking_linux64_csgo.mak#L324
- matchmaking/matchmaking_ds_linux64_csgo.mak#L93
- matchmaking/matchmaking_ds_linux64_csgo.mak#L324
- utils/vpc/vpc/projectgenerator_makefile.cpp#L995

Comment (yumirak) [reply dedicated_main/main.cpp:1]: @r-a-sattarov as far as i know those file are not used, it was leftover from vpc.

Comment (r-a-sattarov) [reply dedicated_main/main.cpp:1]: > @r-a-sattarov as far as i know those file are not used, it was leftover from vpc.

OK. LGTM.

**Review by r-a-sattarov — APPROVED** [review]

File: dedicated_main/main.cpp
Comment (tyabus) [reply dedicated_main/main.cpp:1]: I'm not against VPC support if it's not that much of a hassle.

**Pull request merged**